values.yaml

```diff
@@ -447,6 +453,40 @@ clair:
   ## Additional deployment annotations
   podAnnotations: {}
 
+trivy:
+  # enable is the flag to enable Trivy scanner
+  enabled: true
+  image:
+    # repository the repository for Trivy adapter image
+    repository: goharbor/trivy-adapter-photon
+    # tag the tag for Trivy scanner adapter image
+    tag: dev
+  # replicas the number of Pod replicas
+  replicas: 1
+  # cacheDir is Trivy cache directory
+  # This is where Trivy stores its vulnerabilities DB/index and caches Docker images. The size of the cache might grow
+  # over time and requires housekeeping.
+  cacheDir: "/home/scanner/.cache/trivy"
+  # reportsDir is Trivy reports directory
+  # It's used by the scanner adapter to store a temporary scan report generated by Trivy in JSON format before it's
+  # transformed and stored in Harbor DB.
+  reportsDir: "/home/scanner/.cache/reports"
```

Comment:
Seems that configuring the `cacheDir` and `reportsDir` for users is meaningless as they are both the directories inside the container

Reply:
That's a good point. I'll remove it.